includes/admin/class-wc-admin-addons.php

```diff
@@ -332,30 +332,32 @@ public static function output_wcs_banner_block( $block = null ) {
 		);
 
 		$defaults = array(
-			'image'       => WC()->plugin_url() . '/assets/images/wcs-extensions-banner.png',
+			'image'       => WC()->plugin_url() . '/assets/images/wcs-extensions-banner-3x.png',
 			'image_alt'   => __( 'WooCommerce Services', 'woocommerce' ),
-			'title'       => __( 'Transform your store into a shipping and fulfillment machine', 'woocommerce' ),
-			'description' => __( 'WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.', 'woocommerce' ),
+			'title'       => __( 'Buy discounted shipping labels, print them from your dashboard.' ),
```

Comment:
> Should this be:

> Buy discounted shipping labels AND print them from your dashboard.

I think it _could_ be that. cc @olaolusoga and @MarinaWoo for word-smithing input.